api/src/main/java/org/pmiops/workbench/cdr/CdrVersionService.java

```diff
@@ -1,55 +1,50 @@
 package org.pmiops.workbench.cdr;
 
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
 import java.util.List;
 import java.util.Optional;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
 import javax.inject.Provider;
-import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.access.AccessTierService;
+import org.pmiops.workbench.db.dao.AccessTierDao;
 import org.pmiops.workbench.db.dao.CdrVersionDao;
+import org.pmiops.workbench.db.model.DbAccessTier;
 import org.pmiops.workbench.db.model.DbCdrVersion;
-import org.pmiops.workbench.db.model.DbStorageEnums;
 import org.pmiops.workbench.db.model.DbUser;
 import org.pmiops.workbench.exceptions.ForbiddenException;
 import org.pmiops.workbench.firecloud.FireCloudService;
+import org.pmiops.workbench.model.CdrVersionListResponse;
+import org.pmiops.workbench.model.CdrVersionMapResponse;
+import org.pmiops.workbench.model.CdrVersionMapResponseInner;
 import org.pmiops.workbench.model.DataAccessLevel;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class CdrVersionService {
+  private final AccessTierDao accessTierDao;
+  private final AccessTierService accessTierService;
+  private final CdrVersionDao cdrVersionDao;
+  private final CdrVersionMapper cdrVersionMapper;
+  private final FireCloudService fireCloudService;
+  private final Provider<DbUser> userProvider;
 
-  private static final ImmutableSet<Short> REGISTERED_ONLY =
```

Comment:
Access requirements for tiers are now independent instead of hierarchical.

Also, PROTECTED is not a thing that actually exists